project/.github/workflows/release.yml.jinja

```
@@ -0,0 +1,25 @@
+name: release
+
+on: push
+permissions:
+  contents: write
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/')
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Fetch all tags
+      run: git fetch --depth=1 --tags
+    - name: Setup Python
+      uses: actions/setup-python@v4
+    - name: Install git-changelog
+      run: pip install git-changelog
+    - name: Prepare release notes
+      run: git-changelog --release-notes > release-notes.md
```

```suggestion
      run: git-changelog --release-notes >> release-notes.md
```
I know I'm getting into real edge cases here, but I think this allows some sort of extendability.

---

I prefer staying explicit about our intent: the workflow is configured to only write the output of the `git-changelog` command into the release notes, and nothing else. Using `>>` while not actually making use of it makes it seem like something else could have been writing to `release-notes.md` before, which is not supposed to be the case.

Users can always customize that in their generated projects :slightly_smiling_face:

---

Sure, I keep forgetting that copier honors local changes. What a bliss!
Thanks for your feedback on this :)